src/Microsoft.AspNetCore.Blazor.Build/Cli/Commands/BuildIndexHtmlCommand.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.AspNetCore.Blazor.Build.Cli.Commands
+{
+    internal class BuildIndexHtmlCommand
+    {
+        public static void Command(CommandLineApplication command)
+        {
+            var clientPage = command.Option("-hp|--html-page",
+                "Path to the HTML Page containing the Blazor bootstrap script tag.",
+                CommandOptionType.SingleValue);
```

Comment:
Since this tool isn't intended to be invoked by developers directly, we don't need to do nice things like have shorthand aliases for the options. For simplicity I'd prefer not to have extra ways of doing the same thing.

Reply:
Sure